src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('x is %d, y is %f, z is "%s"'%(x, round(y, 2), z))
```

Comment:
`round` works, but it changes the underlying value of y. Can we do this without altering the original value?